api/autonomous_car/autonomous_car.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import random
+
+DEFAULT_CAR_ROUTE_FILENAME = 'successful_car_route_no_obstacles.json'
+
+CAR_ROUTE_FILENAMES = [
+    DEFAULT_CAR_ROUTE_FILENAME,
+    'successful_car_route_with_obstacles.json',
+    'failure_car_route_out_of_bounds.json',
+    'empty_car_route.json',
+]
+
+ROUTE_DIRECTORY = os.path.join(os.path.dirname(__file__), 'fixtures')
+
+
+def _open_config(filename):
+
+    with open(filename) as config_file:
+        json_contents = json.load(config_file)
+
+    return json_contents
+
+
+def get_car_route(get_random=False):
```

Comment:
Can you restructure this to be more explicit? 
i.e. 
```
def get_car_route(route_name): 
```
and another function to get the random car route
```
def get_random_route(): 
```